docs/04_User_Guides/autofix_js_css_resources.md

```diff
@@ -24,14 +18,41 @@ will lint all your css  files and automatically fix the errors
 more info for crafty eslint options here:  https://swissquote.github.io/crafty/Packages/crafty-preset-eslint/index.html
 
 
-# Option2 :  Prettier is used internally with crafty 
+## Option 2 :  integrate crafty linter in idea 
```

Comment:
Actually option 2 and 3 refer to the same thing.
1. You have to write the configuration file
2. You use prettier to write the files (using the configuration written in 1)

Reply:
I merged the two options and in beginning I am explaoing that `yarn crafty ide` will generate prettier configuration. If there is some other page  more details we can add it. Or add a more in depth guide for yarn crafty ide.

Reply:
There is already a complete section about it : https://swissquote.github.io/crafty/IDE_Integration.html

Reply:
added the link :)